charts/parca/values.yaml

```diff
@@ -14,7 +14,7 @@ agent:
     # -- Overrides pullpolicy
     pullPolicy: IfNotPresent
     # -- Overrides the image tag
-    tag: "v0.8.0"
+    tag: "v0.8.2"
```

Comment:
We also have https://github.com/parca-dev/parca-agent/releases/tag/v0.9.0-rc.0
Possibly we'll have v0.9.0-rc.1

Reply:
if they're gonna be released at same time, then yeah, can be done

Reply:
We'll release both at the same time.

https://github.com/parca-dev/parca-agent/releases/tag/v0.9.0-rc.1 released